### PR TITLE
Remove gcp and cloud region from kafka create

### PR DESCRIPTION
**Pull request by dimakis**
Closes # 

### Verification Steps

1. Do x
2. Do y

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,1083 +1,4 @@
 
-<a name="v0.53.0"></a>
```
[the remark this answers is not in the document]
Comment (dimakis) [reply CHANGELOG.md]: yes, my bad.  I should have noticed that. thank you Rama

File: pkg/cmd/kafka/openshift-cluster/openshiftclustercmdutil/dedicated_util.go [hunk 2]
```diff
@@ -42,15 +42,18 @@ func (v *Validator) ValidatorForMachinePoolNodes(val interface{}) error {
 	return nil
 }
 
-func CreatePromptOptionsFromClusters(clusterList *kafkamgmtclient.EnterpriseClusterList, clusterMap *map[string]v1.Cluster) *[]string {
+func CreatePromptOptionsFromClusters(clusterList *kafkamgmtclient.EnterpriseClusterList, clusterMap *map[string]v1.Cluster) (*[]string, error) {
 	promptOptions := []string{}
 	validatedClusters := ValidateClusters(clusterList)
+	if len(validatedClusters.Items) == 0 {
+		return &promptOptions, fmt.Errorf("no registered OpenShift clusters found, an OpenShit cluster must be registered to use the service ")
```
Comment (rkpattnaik780) [inline pkg/cmd/kafka/openshift-cluster/openshiftclustercmdutil/dedicated_util.go:49]: nit: Will it be tough to use the localizer here?

**Review by rkpattnaik780 — APPROVED** [review]
Verified from a trial only and standard only (with only standard billing model quota) account.

File: pkg/cmd/kafka/openshift-cluster/openshiftclustercmdutil/dedicated_util.go (hunk 2 above)
Comment (dimakis) [reply pkg/cmd/kafka/openshift-cluster/openshiftclustercmdutil/dedicated_util.go]: great catch. I've updated it to use localizer 👍

**Pull request merged**